lighting-estimation-explainer.md

```diff
@@ -5,79 +5,130 @@ This document explains the portion of the WebXR APIs that enable developers to r
 
 "Lighting Estimation" is implemented by AR platforms using a combination of sensors, cameras, algorithms, and machine learning.  Lighting estimation provides input to rendering algorithms and shaders to ensure that the shading, shadows, and reflections of objects appear natural when presented in a diverse range of settings.
 
-The XRLightProbe and XRReflectionProbe interfaces expose the values that the platform offer to WebXR rendering engines.  Their corresponding accessor functions, XRFrame.getGlobalLightEstimate() and XRFrame.getGlobalReflectionProbe() are only accessible once the first frame of an AR session has started.  The promises may be resolved on the same frame or multiple frames later, depending on the platform capabilities.  In some cases, the promises may fail, indicating that the lighting values are not available at this time and should be requested again at a later time.
+The `XRLightProbe` interface exposes the values that the platform offer to WebXR rendering engines. The corresponding retrieval method, `XRSession.requestLightProbe()`, is only accessible once the an AR session has started and returns a promise. The promise may be resolved on the same frame or multiple frames later, depending on the platform capabilities. In some cases, the promise may fail, indicating that the lighting values are not available for that session.
```

Comment:
“accessible once **the an** AR session...“

Reply:
Fixed, thanks!